.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
Tracking the `.envrc` in version control is debated; personally I am in favor of it

Reply:
Personally I would prefer not to commit to this, since a lot more people have direnv than Nix, and Nix isn't required to easily develop on this repo.